src/business_logic/fact_state/in_memory_state_reader.rs

```diff
@@ -146,4 +146,17 @@ mod tests {
             Ok(contract_class)
         )
     }
+
+    #[test]
+    #[should_panic]
+    fn count_actual_storage_changes_should_panic() {
```

Comment:
This happens because of a `todo!` right? Maybe we could rename it as `count_actual_storage_changes_is_a_wip`, or add a comment on why it panics.